cdap-security/src/test/java/io/cdap/cdap/security/authorization/DefaultAccessEnforcerTest.java

```diff
@@ -276,7 +286,7 @@ public void testIsVisibleWithEncryptedCredential()
            new AccessControllerInstantiator(CCONF, AUTH_CONTEXT_FACTORY)) {
       AccessController accessController = accessControllerInstantiator.get();
       DefaultAccessEnforcer accessEnforcer = new DefaultAccessEnforcer(CCONF, sConfCopy, accessControllerInstantiator,
```

Comment:
nit: move to setup / factory method. This way if we need to change constructor we don't need to update in 10 places